x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/helpers.ts

```diff
@@ -133,23 +138,21 @@ export function getDisallowedTermsMessage(
         );
         if (!activeDataFieldNameMatch || currentTerms.length === 0) {
           if (fieldNames.length === 1) {
-            const response: FieldStatsResponse<string | number> = await core.http.post(
-              `/api/lens/index_stats/${indexPattern.id}/field`,
-              {
-                body: JSON.stringify({
-                  fieldName: fieldNames[0],
-                  dslQuery: buildEsQuery(
-                    indexPattern,
-                    frame.query,
-                    frame.filters,
-                    getEsQueryConfig(core.uiSettings)
-                  ),
-                  fromDate: frame.dateRange.fromDate,
-                  toDate: frame.dateRange.toDate,
-                  size: currentColumn.params.size,
-                }),
-              }
-            );
+            const currentDataView = await data.dataViews.get(indexPattern.id);
+            const response: FieldStatsResponse<string | number> = await loadFieldStats({
```

Comment:
How to test changes to `fixAction` in Lens UI?

Reply:
Configure a chart with top values and dimensions with two different time shifts. Instead of a chart a "fix it" action is shown. I just tested, seems to work fine